packages/eslint-plugin/src/rules/use-unknown-in-catch-callback-variable.ts

```diff
@@ -0,0 +1,356 @@
+import type { TSESLint, TSESTree } from '@typescript-eslint/utils';
+import { AST_NODE_TYPES } from '@typescript-eslint/utils';
+import type {
+  ReportDescriptor,
+  Scope,
+} from '@typescript-eslint/utils/ts-eslint';
+import * as tsUtils from 'ts-api-utils';
```

Comment:
[Naming] Everywhere else lowercases this:

```suggestion
import * as tsutils from 'ts-api-utils';
```

Reply:
changed